Translated Demo and PHP Server Demo

JavierCanon opened the pull request:
Hello, please check if its ok.

Thanks.

PoLaKoSz commented on README.md:
```diff
@@ -1,11 +1,10 @@
-# Exception Reporter.NET
+# Shark.NET Error Reporter 
```
I think You didn't wanted to commit this.

JavierCanon commented:
@PoLaKoSz:
i don't think so, if you check the structure of project, for libraries is only copy and paste the new versions, and maybe run and check for changes that breaks the demo. Specially for newbies, you want a click a run demo, no download and setup many things for test if a project fits your needs.  At this time the example working with curl and not with the library, check the php code and comments. I think to, that the webservice that you are building, maybe is better an example not a part of library. 

I changed the name but i fixed later, because i don't wanna confusing the target of library (.net 4.7.2) of my fork.  Im learning to use Github

PoLaKoSz commented:
I agree with You on *everybody like simple working demo* (**including me**) but if I want to test this out I would just use the SMTP option and send it to my email address. :smile:

I don't thing a newbie will complicate his / her life to send it to a webpage than do things with it (and send it to some email address). I think this is a bit advanced scenario.

PandaWood commented:
BTW have you seen the [WebService demo](https://github.com/PandaWood/ExceptionReporter.NET/tree/master/src/WebService.ExceptionReporter) I added recently?

You just hit F5 (after setting dual startup projects) and it runs a REST WebService - fully functional.
Your default browser even opens at the correct URL to retrieve data sent to it.

It's written in .NET Core (2.1) so there are no additional libraries, just a few files with a few lines of code.
https://github.com/PandaWood/ExceptionReporter.NET/tree/master/src/WebService.ExceptionReporter

The demo already has the config pointing to the correct localhost URL for `ResportSendMethod.WebService`

PandaWood commented:
Nevertheless, thanks for this and i think I'm going to pull or copy this into another project just for demos - eg https://github.com/PandaWood/ExceptionReporter.NET-Demos
I might rename that still but it will be a place for all server-related demos
It would be better if I could work out a way to include the libraries you need from a nuget package or similar

PandaWood commented:
Thanks for this, I'm not going to include these PHP demos etc in this repository, but I'm planning to copy these over to another (demo) repository - probably this one https://github.com/PandaWood/ExceptionReporter.NET-Demos
